web/styles/dark_theme.css

```diff
@@ -510,19 +510,11 @@
     }
 
     .recent_view_container #recent_view_table .button-recent-filters {
-        background-color: hsl(211deg 29% 14%);
-        border-color: hsl(0deg 0% 0%);
-        color: hsl(0deg 0% 100%);
-
         &:focus {
             background-color: hsl(0deg 0% 0% / 50%) !important;
-            outline: 0;
         }
 
         &.fake_disabled_button {
-            cursor: not-allowed;
-            opacity: 0.5;
-
             &:hover {
                 background-color: hsl(0deg 0% 0% / 50%);
                 border-color: hsl(0deg 0% 0%);
```

Comment:
Probably a good follow-up would be to make the 3 remaining colors defined here use CSS variables and be able to delete this block from `dark_theme.css`, while we're touching it.

Reply:
Added to my TODO.